eng/Version.Details.xml

```diff
@@ -48,9 +48,9 @@
       <Uri>https://github.com/dotnet/symreader-converter</Uri>
       <Sha>c5ba7c88f92e2dde156c324a8c8edc04d9fa4fe0</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.XliffTasks" Version="1.0.0-beta.23360.1" CoherentParentDependency="Microsoft.DotNet.Arcade.Sdk">
+    <Dependency Name="Microsoft.DotNet.XliffTasks" Version="1.0.0-beta.23516.1" CoherentParentDependency="Microsoft.DotNet.Arcade.Sdk">
       <Uri>https://github.com/dotnet/xliff-tasks</Uri>
-      <Sha>a171b61473272e5a6d272117963864ba958a012a</Sha>
+      <Sha>d66a09c8b11e1235273c8e6cd946e5c8e247f189</Sha>
       <SourceBuild RepoName="xliff-tasks" ManagedOnly="true" />
     </Dependency>
```

Comment:
@Evangelink I don't have permissions in this repository. Please update these lines similar to what I mentioned in the roslyn comment here: https://github.com/dotnet/roslyn/pull/70597/files#r1375319003

Use the latest version and sha from the above Arcade.Sdk entry.

(same as in test-templates)

BTW, you will also need to resolve the merge conflicts.

Reply:
Done, thanks!